app/carpool/views.py

```diff
@@ -77,29 +77,19 @@ def start_geojson():
         pools = pools.filter(Carpool.leave_time >= datetime.datetime.utcnow())
 
     try:
-        near_lat = request.args.get('near.lat', type=float)
-        near_lon = request.args.get('near.lon', type=float)
+        near_lat = request.args.get('near.lat', None, type=float)
```

Comment:
`None` is the default return value already. You shouldn't have to specify it here.